training/culture-and-process/DIGITAL_PUBLISHING.md

```diff
@@ -1,32 +1,61 @@
 
+## Digital Publishing
 
+Digital Publishing is the arm of the ONS whose primary focus is on providing content through digital means. The main areas of operation involve the ONS website, API, developer site, content management system, social media, digital and print design, data visualisation, content and editorial work, and various other “digital” and “publishing” things.
 
-Further resources
-----------------------------
+The main aims of Digital Publishing can be summarised as the following:
 
-Links to other resources, and what they're useful for explaining.
-May or may not be materials that extend directly from the course
\ No newline at end of file
+- Continuously improve the user experience of our website, including how we publish open data
+- Future proof the organisation’s digital publishing capability
+- Develop new digital products that extend the impact of our data and analysis
+- Support publishing operations and policy
+- Understand our online communities to make sure we are driven by user needs
+- Measure the impact of our online presence
+
+Digital Publishing is made up of a number of different departments and sub-departments. The three main areas that these departments cover are: Publishing and Content, Development, and Insight.
+
+### Publishing and Content
+
+The Publishing and Content area covers content design and publishing operations. This includes a team of publishing officers and content editors, who are responsible for the timely and accurate dissemination of official statistics, and helping to develop and maintain digital and content standards.
+
+Content design is a multi-disciplinary team of data journalists, data visualisation specialists, design and social media experts whose primary goal is to produce content for the ONS.
+
+### Development
+
+The Development area focuses mainly on the software development aspect of Digital Publishing, with software teams working on a range of projects, from improving and advancing existing products, through to creating new systems and tools that will transform how the organisation works. 
+
+Work within this department generally involves the ONS website, content management systems, and publishing platforms. More specific focus can be on the API and backend, which is responsible for our public-facing APIs, our developer site, and our backend publishing systems. Additional focus is on our data import and export pipelines, and working with statistical teams to migrate their data, and also exploring ideas we may take through to production in the future.
+
+Development areas also include Dev Ops, Interaction Design, and Data Delivery. 
```

Comment:
The devops side of this might be worth bringing out a bit more - to make it clear that our developers are responsbile for all of our platform work and the maintenance of our applications, rather than potentially imply that we have specific Dev Ops people other than the devs.